src/medCore/gui/toolboxes/medSegmentationAbstractToolBox.h

```diff
@@ -26,13 +26,18 @@ class MEDCORE_EXPORT medSegmentationAbstractToolBox : public medToolBox
     Q_OBJECT
 
 public:
-             medSegmentationAbstractToolBox(QWidget *parent = 0);
+    medSegmentationAbstractToolBox(QWidget *parent = 0);
     virtual ~medSegmentationAbstractToolBox();
 
     virtual dtkPlugin* plugin() = 0;
 
     virtual medAbstractData *processOutput() = 0;
 
+public slots:
+
+    //! Launched when the view is updated
```

Comment:
I think the name of the function is sufficient here. These sort of comments pollute the code.